### DISCUSSION TITLE
Renewing Let's encrypt certificate on a Digital Ocean droplet

### DISCUSSION THREAD

**Discussion by DidierMalenfant** [owncast/owncast#2856] [Q&A] [answered]
I created my owncast instance using the Digital Ocean template which was awesome to get up and running quickly.

I just got an email notification that my SSL certificate is about to expire and I only now realize it seems like that template doesn't include certbot to auto renew the certificate.

How does everyone handle renewal? Manually? Automatically? I'm not very well versed in all this stuff so if anyone can point to an easy step by step procedure that would be great. Thanks!

**Comment by gabek** [comment]
Caddy should auto-renew your Let's encrypt certificate. You could probably manually force it to renew, however. [This thread](https://caddy.community/t/how-to-force-renewal-of-lets-encrypt-certificates/14843/17) seems to discuss that.

**Comment by DidierMalenfant** [comment]
Yeah I also found that thread and it seems to discuss that but... it doesn't.

The only info is:
```
You can force renewal by deleting the record in the storage then reloading or restarting Caddy.
```
but that doesn't help as to how this is actually achieved, where the storage is, etc...

I'll just wait and see I guess since it looks like it's just the renewal threshold not being set correctly in caddy and causing the warning email.

**Reply by gabek** [reply]
This might be helpful in that case: https://caddy.community/t/where-does-caddy-keep-their-certificates/4728/3

**Reply by DidierMalenfant** [reply]
First line of that post is a fail for me... `/usr/local/etc/rc.d/caddy` does not exist in my install. It's ok, I'm not looking for help googling though. I'd found all those articles before.

I was hoping that caddy was part of a fairly typical owncast install and that someone here would have some experience with this and could help me with a clear step by step solution.

I realize that this is out of the scope for owncast so I won't bother you any further.

**Comment by DidierMalenfant** [comment, marked as answer]
Turns out, Caddy had renewed the certificate but via another issuer (ZeroSSL in my case) and so let's encrypt had no idea this had taken place. All is good then...

https://caddy.community/t/let-s-encrypt-certificate-expiring-soon-with-no-indication-caddy-will-renew-it/19521

**Reply by gabek** [reply]
Makes complete sense. Thanks for the update!
